### PR TITLE
Editorial: clarify Permissions Policy / Powerful Feature overlap

### PR DESCRIPTION
**Pull request by marcoscaceres**
Based on recent discussions, we note that most (but not all!) powerful features are also policy-controlled features.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/permissions/pull/367.html" title="Last updated on Feb 25, 2022, 12:47 AM UTC (d122c3d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/permissions/367/4c4eebc...d122c3d.html" title="Last updated on Feb 25, 2022, 12:47 AM UTC (d122c3d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -240,13 +240,35 @@ <h3>
...
+        <p data-cite="permissions-policy">
           A <dfn class="export" data-local-lt="feature">powerful feature</dfn> is a web platform
           feature (usually an API) for which a user gives [=express permission=] before the feature
-          can be used. Access to the feature is determined by the user having
-          [=permission/granted=] permission, or by satisfying some criteria that is equivalent to a
-          permission [=permission/grant=].
+          can be used. Except for a few notable exceptions (e.g., the [[[Notifications]]]), most
+          powerful features are also [=policy-controlled features=]. For powerful features that are
+          [=policy-controlled features=], [[[permissions-policy]]] controls whether a [=document=]
+          is [=allowed to use=] a feature, and only if it is allowed can a powerful feature request
```
Comment (kenchris) [inline index.html:249]: "and only if it is allowed can a powerful feature request" this sounds a bit weird to me, maybe move "can" to the end

File: index.html [hunk 2]
```diff
@@ -240,13 +240,35 @@ <h3>
...
+          user having [=permission/granted=] permission, or by satisfying some criteria that is
+          equivalent to a permission [=permission/grant=].
         </p>
+        <aside class="example" title="Powerful features are policy-controlled features">
+          <p>
+            This example shows how the permissions policy set through the [^iframe/allow^]
+            attribute controls whether the [^iframe^] is [=allowed to use=] a powerful feature.
+            Because `"geolocation"` is allowed, the [^iframe^]'s document can request permission
+            from the user to use the [[[Geolocation]]] (i.e., it will prompt the user for express
+            permission to access their location information). However, request permission to use
```
Comment (miketaylr) [inline index.html:260]:
```suggestion
            permission to access their location information). However, requesting permission to use
```

File: index.html (hunk 1 above)
Comment (miketaylr) [reply index.html:249]: A couple of possible rewording suggestions:

1. For powerful features that are also policy-controlled features, Permissions Policy controls whether a document is allowed to use a given feature. A powerful feature may only request express permission from a user if the document has permission delegated to it via the corresponding policy-controlled feature.

2. For powerful features that are also policy-controlled features, a powerful feature may only request express permission from a user if and only if a document has permission delegated to it via Permissions Policy.

**Review by miketaylr — APPROVED** [review]
LGTM w/ nits

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html]: Went with option 1, with a small clarification.

**Pull request merged**